### DISCUSSION TITLE
Change the documentation to the wiki

### DISCUSSION THREAD

**Discussion by TeoDev1611** [wbthomason/packer.nvim#541] [Ideas]
For a best experience and documentation can you migrate from the README to the Wiki

**Comment by wbthomason** [comment]
I'd be happy to review and merge PRs cleaning up the docs, but I think that most things should remain in both the README and the Vimdocs. Is there a particular argument for the Wiki being preferable? I find that it is less discoverable than the aforementioned other two options.

**Reply by TeoDev1611** [reply]
The wiki is the best way for me why this section is made for this 😅.  If you want to migrate you can just create the first page and I can help migrate the documentation to the wiki :)

**Reply by 0oAstro** [reply]
Hey I will also be happy to migrate docs to wiki :)

**Reply by wbthomason** [reply]
Thanks for the offers. I'm still not convinced that the wiki is better than just cleaning up the README and Vimdocs, but I've created the initial wiki page to enable you to migrate the documentation if you'd like.
